src/app/util/attribute-storage.cpp

```diff
@@ -963,11 +967,6 @@ bool emberAfEndpointEnableDisable(EndpointId endpoint, bool enable)
                                                app::Clusters::Descriptor::Attributes::PartsList::Id);
     }
 
-    if (!enable)
-    {
-        emAfEndpoints[index].bitmask.Clear(EmberAfEndpointOptions::isEnabled);
```

Comment:
This should really come after `shutdownEndpoint`, I would think... So maybe we need to break up the `currentlyEnabled != enable` block into two blocks?

Reply:
Okay, I see the problem, so maybe let's move it and put it just after `shutdownEndpoint` within the same `else` block?